pkg/server/share_manager.go

```diff
@@ -210,6 +240,96 @@ func (m *ShareManager) resizeVolume(devicePath, mountPath string) error {
 	return nil
 }
 
+func (m *ShareManager) makeLeaseClient() error {
+	cfg, err := rest.InClusterConfig()
+	if err != nil {
+		m.logger.WithError(err).Warn("Failed to create InClusterConfig")
+		return err
+	}
+
+	clientset, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		m.logger.WithError(err).Warn("Failed to create kubernetes client set")
```

Comment:
This error is logged at the upper layer. We should simply return it here.

Reply:
Done, thanks.